docs/example-manifests/README.md

```diff
@@ -0,0 +1,15 @@
+# Example Manifests
+
+The following directory provides examples of static pod, daemonset and systemd manifests
+that were used in a working Kubernetes cluster on DigitalOcean. Use the following examples as a guide
+to setup your cluster correctly using the DigitalOcean Cloud Controller Manager.
+Note that the examples are not meant to be copy and pasted as it contains configuration
+for a specific cluster.
```

Comment:
I think this is so they can set up a Kubernetes cluster? If that is the case, do we really need to take care of that aspect as well? Otherwise, I think we only should provide the `cloud-controller-manager.yml` file to the user.

Reply:
In general, I agree that it's not needed. But for now just following requirements set by https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0019-cloud-provider-documentation.md#requirement-1-1 which assumes some people may choose to manage control plane themselves and want examples of how to configure them for DigitalOcean.

Reply:
Gotcha